vulnfeeds/cves/versions.go

```diff
@@ -257,6 +257,78 @@ func ExtractVersionInfo(cve CVEItem, validVersions []string) (VersionInfo, []str
 	return v, notes
 }
 
+func ExtractVersion(cve CVEItem) (VersionInfo, []string) {
```

Comment:
As discussed, rather than duplicating all of these, let's just change the existing `ExtractVersionInfo` function to allow passing an empty `validVersions`.

Reply:
I've completely excised the call to this because we're focusing on commits and not versions, so this need is now removed.